Core/Utilities/SitesDIRACGOCDBmapping.py

```diff
@@ -6,26 +6,28 @@
 __RCSID__ = "$Id$"
 
 from DIRAC import gLogger, gConfig, S_OK, S_ERROR
-from DIRAC.ConfigurationSystem.Client.Helpers.Path import cfgPath
+from DIRAC.DataManagementSystem.Utilities.DMSHelpers import DMSHelpers
+from DIRAC.ResourceStatusSystem.Utilities.CSHelpers import getSEHost
```

Comment:
This isn't the best location from where to pick this up, now. Any advice on where to move it and get it from?

Reply:
The `DIRAC.ResourceStatusSystem.Utilities.CSHelpers ` location is... not the best (the module was created long time ago before, e.g., the `ResourcesHelper` came to life. So a module in `/Core` like this one shouldn't probably use it.
So the question is: where should I move the `getSEHost` method?

Reply:
DMSHelper ?

Reply:
Alright, I'll move it. promised. But right now I merge it, I need this now...